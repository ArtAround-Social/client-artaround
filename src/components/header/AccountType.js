import React from 'react';
import PropTypes from 'prop-types';

function AccountType({ user_type }) {
  return(
    <h3>{user_type} Account</h3>
  );
}

AccountType.propTypes = {
  user_type: PropTypes.string.isRequired
};

export default AccountType;
