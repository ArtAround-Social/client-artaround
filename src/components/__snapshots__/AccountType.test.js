import React from 'react';
import { shallow } from 'enzyme';
import AccountType from '../header/AccountType';


describe('AccountType component', () => {
  it('renders account type', () => {
    const wrapper = shallow(
      <AccountType
        user_type = 'Artist'
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
