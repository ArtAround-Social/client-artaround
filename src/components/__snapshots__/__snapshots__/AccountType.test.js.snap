// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountType component renders account type 1`] = `
<h3>
  Artist
   Account
</h3>
`;
